test/caption-stream-test.js

```diff
@@ -216,6 +216,48 @@
     }, 'parsed the third caption');
   });
 
+  test('backspaces are applied to non-displayed memory', function() {
+    var captions = [];
+    cea608Stream.on('data', function(caption) {
+      captions.push(caption);
+    });
+
+    // RCL, resume caption loading
+    cea608Stream.push({ ccData: 0x1420 });
+    // '01'
+    cea608Stream.push({
+      ccData: ('0'.charCodeAt(0) << 8) | '1'.charCodeAt(0)
```

Comment:
If I put the shift-ORing behind a function, you feel that would make the resulting test clearer? I thought this was sufficiently terse enough that it made it clear how `ccData` works without adding too much noise to the tests.